packages/blockchain-link/package.json

```diff
@@ -67,6 +67,7 @@
         "worker-loader": "^3.0.8"
     },
     "dependencies": {
+        "@solana/buffer-layout": "^4.0.1",
```

Comment:
hmm good for feat/solana but might be questioned later. can't we handle this with current deps?